Sources/AppStoreConnectCLI/Model/API/ExtendedBetaGroup.swift

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+
+struct ExtendedBetaGroup {
+    let app: AppStoreConnect_Swift_SDK.App
+    let betaGroup: AppStoreConnect_Swift_SDK.BetaGroup
+}
```

Comment:
I wonder if it is possible to not have another model to achieve this? This is a little bit confusing, but no big deal

Reply:
At first I tried using a tuple but I found it was cleaner to make an intermediary model